usb_manager.plg

```diff
@@ -134,9 +134,9 @@ Revert Changes for 2021.12.12
 
 - Initial beta release.
 
-If you are using USBIP-GUI continue to do so at this time. This plugin will suppercede USBIP-GUI in the future and will migrate configurations.
+If you are using USBIP-GUI continue to do so at this time. This plugin will supercede USBIP-GUI in the future and will migrate configurations.
 
-USBIP-GUI and USB_Manager cannot co-exist. If you want to replace USBIP-GUI then uninstall first, Config files we remane on the flash drive you can copy them to usb_manager directory.
+USBIP-GUI and USB_Manager cannot co-exist. If you want to replace USBIP-GUI then uninstall first, Config files we rename on the flash drive you can copy them to usb_manager directory.
```

Comment:
It's unclear to me whether the intention was to say "we rename" or "will remain".
There should be punctuation or some linking vocabulary between "drive you" as well

Reply:
Thanks for the PR will review.